tests/unit/amo/reducers/test_collections.js

```diff
@@ -444,6 +451,24 @@ describe(__filename, () => {
       expect(collectionInState.addons)
         .toEqual(createInternalAddons(addons));
     });
+
+    it('loads notes for collection add-ons', () => {
+      const notes = 'These are some notes.';
+      const fakeCollectionAddon =
+        createFakeCollectionAddon({ notes });
```

Comment:
These two lines should fit on a single line, no?

Reply:
This has not been changed.

Reply:
I thought I changed that. Odd. I'll fix it.